.gitignore

```diff
@@ -5,3 +5,4 @@ __pycache__/
 /build/
 /dist/
 /tmp/
+spdx/parsers/parsetab.py
```

Comment:
What was the rationale to remove this file entirely? I know it is generated, but was removing it because of Python3 changes?